fluid/vgg16.py

```diff
@@ -152,10 +167,11 @@ def test(exe):
                   (pass_id, iters, loss, acc))
         pass_elapsed = time.time() - start_time
 
-        pass_test_acc = test(exe)
+        pass_test_acc = eval_test(exe, accuracy, avg_cost)
+        pass_acc = accuracy.eval(exe)
```

Comment:
Maybe this is not right. 
It should be :
```
pass_acc = accuracy.eval(exe)
pass_test_acc = eval_test(exe, accuracy, avg_cost)
```
Because you rested `accuracy` in `eval_test `.